cmd/registry/conformance/linter.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package conformance
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/apigee/registry/log"
+	"github.com/apigee/registry/rpc"
+	"google.golang.org/protobuf/proto"
+)
+
+type ruleMetadata struct {
+	guidelineRule *rpc.Rule      //Rule object associated with the linter-rule
+	guideline     *rpc.Guideline //Guideline object associated with the linter-rule
```

Comment:
These comments don't look like the others in this file. Specifically 1) missing the space after `//` and 2) not written as a full sentence with a period at the end.

Reply:
Done